rd-cpp/src/rd_framework_cpp/src/main/wire/SocketWire.cpp

```diff
@@ -400,7 +407,7 @@ void SocketWire::Base::ping() const
 	}
 	catch (std::exception const& e)
 	{
-		logger->debug("{}: exception raised during PING | {}", this->id, e.what());
+		logger->warn("{}: exception raised during PING | {}", this->id, e.what());
```

Comment:
People were asking us to stop logging ping errors to `warn`. Please roll this back; it was done on purpose.